Shared/Extensions/JellyfinAPI/SessionInfo.swift

```diff
@@ -0,0 +1,36 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Foundation
+import JellyfinAPI
+
+extension SessionInfo {
+
+    var playMethodDisplayTitle: String? {
+        guard self.nowPlayingItem != nil,
```

Comment:
Here is a refactor to be a little more Swift-y with optional handling and to be exhaustive.

```swift
guard nowPlayingItem != nil, let playState, let playMethod = playState.playMethod else { return nil }

if let transcodingInfo {

    let isVideoDirect = transcodingInfo.isVideoDirect ?? false
    let hasVideoCodec = transcodingInfo.videoCodec != nil
    let isAudioDirect = transcodingInfo.isAudioDirect ?? false

    if isVideoDirect || hasVideoCodec, isAudioDirect {
        return L10n.remux
    } else if isVideoDirect {
        return PlayMethod.directStream.displayTitle
    }
}

switch playMethod {
case .transcode:
    return PlayMethod.transcode.displayTitle
case .directPlay, .directStream:
    return PlayMethod.directPlay.displayTitle
}
```

I would have also made the following comments:
- use `if let` shorthand instead, where applicable
	- https://www.hackingwithswift.com/swift/5.7/if-let-shorthand
- use implicit `self` instead

Reply:
I can swap that out! Just to confirm, outputting a String (for this instance) and its location on SessionInfo is okay? 

I wasn't in love with the positioning but that logic is wonky IMO vs just using .PlayMethod literally.

Reply:
This is where it belongs, since it uses other data from the session itself.